src/setup.js

```diff
@@ -59,6 +59,15 @@ const setupHooks = () => {
 			};
 		}
 	);
+
+	// Hook to disable caching for specific endpoints.
+	addFilter(
+		'native.disable_caching_endpoints',
+		'gutenberg-mobile',
+		( endpoints ) => {
+			return [ ...endpoints, '/oembed.*videopress.com/i' ];
```

Comment:
I'm wondering if we could just simply disable the caching for all requests to `/oembed` endpoint (i.e. regex pattern. `/oembed\/1\.0\/proxy\?.*/i`). We usually use this endpoint via the `getEmbedPreview` selector that is already caching the response, so probably we don't need to have a second caching layer, WDYT?

Reply:
Sure, I just didn't want to introduce too many changes in this PR and keep it focused on the immediate need. I'm fine with opening that up to all embeds if you think it's safe.

Reply:
If we open it to all embeds, we could do it directly in Gutenberg so we won't need the WP hook and make any changes in GB-mobile.

Reply:
Actually thinking if we want to avoid caching all of the embed routes we can do this in Gutenberg directly.